pytorch_lightning/plugins/environments/lsf_environment.py

```diff
@@ -159,3 +216,6 @@ def _get_main_port() -> int:
             log.debug(f"calculated LSF main port: {port}")
             return port
         raise ValueError("Could not find job id in environment variable LSB_JOBID")
+
+    def __str__(self) -> str:
+        return self._rep
```

Comment:
This is not strictly speaking necessary for the fix. Let's do this in another PR to avoid inconsistencies between environment implementations.